### PR TITLE
VB-945 Add prisoner wing to session template

### PR DESCRIPTION
**Pull request by AledWEvans**
## What does this pull request do?

Add prisoner wing to session template

## What is the intent behind these changes?

To allow configuration data of the prisoner wing to be configured

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/model/entity/prison/PrisonWing.kt [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.ManyToMany
+import javax.persistence.Table
+
+@Entity
+@Table(name = "PRISON_WING")
+data class PrisonWing(
```
Comment (AledWEvans) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/model/entity/prison/PrisonWing.kt:14]: may be this should be SessionWing or AllowedSessionWing

File: src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/SessionTemplateDto.kt [hunk 2]
```diff
@@ -34,6 +34,8 @@ data class SessionTemplateDto(
   val openCapacity: Int,
   @Schema(description = "day of week fpr visit", example = "MONDAY", required = false)
   val dayOfWeek: DayOfWeek?,
+  @Schema(description = "List of prison wings associated with the session template", required = false)
```
Comment (dhirajshettymoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/SessionTemplateDto.kt:37]: If there are none associated this session is available to the whole prison, right? Should we mention this on the comments?

Comment (rj-adams) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/SessionTemplateDto.kt:37]: Can I just confirm that there can be multiple wings which can be associated with the session template entry. So if Wings D & F could have visits at the same time, the PrisonWings value would hold "DF" or similar

Comment (AledWEvans) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/SessionTemplateDto.kt:37]: Yes that's correct @rj-adams

Comment (AledWEvans) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/SessionTemplateDto.kt:37]: @dhirajshettymoj Yes I think it should mention it I will add

**Review by dhirajshettymoj — APPROVED** [review]
Looks good.

**Pull request merged**